ReactWindows/ReactNative.Shared/Bridge/Queue/ActionQueueExtensions.cs

```diff
@@ -45,5 +45,21 @@ public static Task<T> RunAsync<T>(this IActionQueue actionQueue, Func<T> func)
 
             return taskCompletionSource.Task;
         }
+
+        /// <summary>
+        /// Calls a function on a message queue and returns a task to await the response.
+        /// </summary>
+        /// <typeparam name="T">Type of response.</typeparam>
```

Comment:
any reason the typeparam is at the top? seems like it should be a nested tag on the Func parameter...

Reply:
Yeah, copy-pasted by mistake, good catch.